packages/debian/rules

```diff
@@ -10,15 +10,18 @@ export DEB_BUILD_HARDENING = 1
 DPKG_EXPORT_BUILDFLAGS = 1
 include /usr/share/dpkg/buildflags.mk
 DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+PATH := $(HOME)/.rbenv/bin:$(HOME)/.rbenv/shims:$(PATH)
+export PATH
 
 %:
 	dh $@
 
 override_dh_auto_configure:
-	dh_auto_configure -- --with-munin-plugins --enable-mruby
+	dh_auto_configure -- --with-munin-plugins --with-ruby=$(shell rbenv which ruby) --enable-mruby
 
-# disable 'make check'.
 override_dh_auto_test:
+	bash -x ./test/unit/run-test.sh
```

Comment:
Is `bash -x` needed?

Reply:
Yes, It may be helpful when test has failed.